.eslintrc.js

```diff
@@ -21,5 +21,6 @@ module.exports = {
     'react/jsx-props-no-spreading': 'off', // HOCs should have prop spreading
     'react-hooks/rules-of-hooks': 'error', // part of react hooks
     'react-hooks/exhaustive-deps': 'warn', // part of react hooks
+    'linebreak-style': 0,
```

Comment:
eslint kept yelling at me for having LF/CRLF line endings every time I saved the file, so I added this so it would stop yelling at me. I don't think it makes a difference in the code? Every time I commit to git it automatically reformats the line endings to match.

Reply:
Okay that makes sense then!